test/ProgressOnderwijsUtils.Tests/Data/ReadJsonTest.cs

```diff
@@ -107,4 +107,53 @@ public void ReadJson_can_read_all_known_used_column_types_from_the_db()
 
         ApprovalTest.CreateHere().AssertUnchangedAndSave(Encoding.UTF8.GetString(pipe.Reader.ReadAsync().GetAwaiter().GetResult().Buffer));
     }
+
+    enum ReadJsonPocoTestId { }
+
+    sealed record ReadJsonPocoTest : IWrittenImplicitly
+    {
+        public ReadJsonPocoTestId ReadJsonPocoTestId { get; init; }
+        public bool BooleanColumn { get; init; }
+        public int? NumberColumn { get; init; }
+        public string? StringColumn { get; init; }
```

Comment:
maar vooral types zoals byte array, datetime e.d. zijn hier interessant - juist alle dingen die niet normaal in json voorkomen.